style.css

```diff
@@ -253,3 +253,80 @@ h2 {
 .get_button:active.see_button:active {
   border: 3px solid #1a2276;
 }
+
+/* contact form start here */
+.contac_me {
+  background-color: #6070ff;
+  width: 375px;
+  border-top-left-radius: 70px;
+  padding-bottom: 10px;
+  margin-top: -10px;
+  background-image: url("./assets/backgrounds/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+}
+
+.text_area_contac {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  color: #fff;
+}
+
+#contact_title {
+  margin-bottom: 2px;
+  padding-top: 80px;
+  font-size: 40px;
+  line-height: 50px;
+}
+
+#contact_text {
+  font-size: 20px;
+  line-height: 28px;
+  padding-left: 9%;
+  padding-right: 10%;
+}
+
+.form_area_contact {
+  display: flex;
+  flex-direction: column;
+  padding: 30px;
+  margin-top: -35px;
+  gap: 8px;
+  margin-right: 8px;
+  align-items: stretch;
+}
+
+#name_contact {
+  border-radius: 5px;
+  border: 0.001px;
+  padding: 5%;
+}
+
+#email_contact {
+  border-radius: 5px;
+  border: 0.001px;
+  padding: 5%;
+}
+
+#textarea_contact {
+  border-radius: 5px;
+  padding: 5%;
+}
+
+#submit_btn {
+  background-color: #fff;
+  color: #6070ff;
+  width: fit-content;
+  font-weight: 500;
+  border-radius: 5px;
+  padding: 5%;
+  cursor: pointer;
+  padding-top: 15px;
+  margin-top: 10px;
+}
```

Comment:
According to the project requirements, this button should have styling for several interactions and those stylings should follow the template design. You are missing a couple of stylings and the ones you have don't follow the template design. Kindly fix this.

<img width="905" alt="Screen Shot 2022-05-31 at 16 35 56" src="https://user-images.githubusercontent.com/82426682/171288398-9c878dad-ff2e-4ae4-bc8b-c40667f9b329.png">

<img width="744" alt="Screen Shot 2022-05-31 at 16 36 16" src="https://user-images.githubusercontent.com/82426682/171288447-8c257d2c-c09c-49bb-b43a-d91373c787e6.png">